sal/socket_api.h

```diff
@@ -45,7 +45,7 @@ typedef socket_error_t (*socket_bind)(struct socket *socket, const struct socket
 typedef socket_error_t (*socket_str2addr)(const struct socket *socket, struct socket_addr *addr, const char *address);
 typedef socket_error_t (*socket_start_listen)(struct socket *socket, const uint32_t backlog);
 typedef socket_error_t (*socket_stop_listen)(struct socket *socket);
-typedef socket_error_t (*socket_accept)(struct socket *sock, socket_api_handler_t handler);
+typedef socket_error_t (*socket_accept_ptr)(struct socket *sock, socket_api_handler_t handler);
```

Comment:
This was a choice that was due to the creation of the new accept API.  It's not ideal, but I would suggest that any API which follows the new pattern should use the _ptr suffix.